lex/slots/MovieQuote.json

```diff
@@ -29,6 +29,42 @@
         }, 
         {
             "value": "you complete me"
+        }, 
```

Comment:
It seems like the more slot values we add, the less likely it is that Lex picks up everything as an actor when it should be a plot or quote.